ig.ini

```diff
@@ -0,0 +1,12 @@
+[IG]
+; ***********************************************************************************************
+; *                               WARNING: DO NOT EDIT THIS FILE                                *
+; *                                                                                             *
+; * This file is generated by SUSHI. Any edits you make to this file will be overwritten.       *
+; *                                                                                             *
+; * This ig.ini was generated using the template property in config.yaml. To provide your own   *
+; * ig.ini, create an ig.ini file in the ig-data folder with required properties: ig, template. *
+; * See: https://build.fhir.org/ig/FHIR/ig-guidance/using-templates.html#root                   *
+; ***********************************************************************************************
```

Comment:
We can ditch this warning, since it is now not true.

Reply:
https://github.com/SaraAlert/saraalert-fhir-ig/pull/6/commits/e90a101d4dc0c7a2e8219aacd6e9bf7cf6da4d62